sdk/core/azure-core-tracing-opencensus/azure/core/tracing/ext/opencensus_span/__init__.py

```diff
@@ -68,15 +68,15 @@ def span_instance(self):
         """
         return self._span_instance
 
-    def span(self, name="span"):
-        # type: (Optional[str]) -> OpenCensusSpan
+    def span(self, name="span", **kwargs):
```

Comment:
Not a new issue, but I'm a bit surprised that this isn't a `@classmethod`...

Reply:
i think the reason is this is supposed to create a child  span  - so `parent_span.span` is more intuitive than `SpanImplementation.span`? not sure what's the thought process behind it when abstract span was created

Reply:
You can call `@classmethod`s through an instance, though. But that is not a change we can make now...